android/filament-android/src/main/cpp/embind/bind_impl.h

```diff
@@ -0,0 +1,11 @@
+//
+// Created by Nicolas Coderre on 1/8/19.
+//
+
+#pragma once
+
+#include <JavaScriptCore/JSBase.h>
```

Comment:
This file is not necessary, and also not correct.
I call BindToContext from java through JNI interface.